functionals.py

```diff
@@ -1,12 +1,15 @@
+import os
+import sys
 import json
+import errno
 from string import Template
-import os
-from esse.utils import read_json_file, dump_json_file
 
-DIR = os.path.dirname(__file__)
-UNIT_FUNCTIONALS_PATH = os.path.join(DIR, "data/schema/models_directory/pb/qm/dft")
+BASE_DIR = os.path.abspath(os.path.dirname(__file__))
+REL_DIR = "schema/models_directory/pb/qm/dft"
+UNIT_FUNCTIONALS_PATH = os.path.join(BASE_DIR, REL_DIR)
 PROTOTYPE_FILENAME = os.path.join(UNIT_FUNCTIONALS_PATH, "dft_unit_functionals_proto.json")
 UNIT_FILENAME = os.path.join(UNIT_FUNCTIONALS_PATH, "dft_unit_functionals.json")
+DATA_FILENAME = os.path.join(BASE_DIR, "src/py/esse/data", REL_DIR, "dft_unit_functionals.json")
```

Comment:
Since the `schema` directory itself is where the symlinking occurs, `DATA_FILENAME` should no longer be necessary.